files/default/sql/ddl/3.8.0__initial_tables.sql

```diff
@@ -2077,6 +2077,7 @@ CREATE TABLE IF NOT EXISTS `transformation_function` (
                                                          `created` timestamp NULL DEFAULT CURRENT_TIMESTAMP,
                                                          `creator` int(11) NOT NULL,
                                                          `save_type`                         VARCHAR(255)    NOT NULL, 
+                                                         `statistics_argument_names`         VARCHAR(5000),
```

Comment:
since there is "," joining the features, `5004` is better, then you can store 5 features of max length